package.json

```diff
@@ -27,13 +27,14 @@
     "@babel/polyfill": "^7.6.0",
```

Comment:
How do you manage to change the env to dev or test?
You can install `cross_env`
Your test script can look like `cross-env NODE_ENV=test nyc mocha...`